src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,194 @@
+package duke;
+
+import java.io.FileNotFoundException;
+import java.util.Scanner;
+import java.util.ArrayList;
+
+import duke.command.DukeException;
+import duke.task.*;
+
+import java.nio.file.Paths;
+import java.nio.file.Files;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class Duke {
+    public static void main(String[] args) throws IOException{
+        String logo = "____________________________________________________________\n"
+                + "Hello! I'm duke.Duke\n"
+                + "What can I do for you?\n"
+                + "____________________________________________________________\n";
+
+        System.out.println(logo);
+
+        ArrayList<Task> todo = new ArrayList<>(100);
+        int todo_index = 0;
+
+        //create duke.txt in folder named 'data'
+        //idea from https://stackoverflow.com/questions/9658297/java-how-to-create-a-file-in-a-directory-using-relative-path
+        File dir = new File("data");
+        dir.mkdirs();
+        File tmp = new File(dir, "duke.txt");
+        tmp.createNewFile();
+        String save = "";
+
+        Scanner sc = new Scanner(System.in);
+        String input = sc.nextLine();
+
+        while (true) {
+            if (input.startsWith("done")) {
+                int i = Integer.parseInt(input.substring(5));
+                todo.get(i - 1).markAsDone();
+                System.out.println("____________________________________________________________");
+                System.out.println("Nice! I've marked this task as done: ");
+                System.out.println(todo.get(i - 1).toString());
```

Comment:
The sole purpose of print statements is to print something. Avoid doing operations like fetching elements in the print statement itself. It would be better to separate this out into different lines.